### DISCUSSION TITLE
Replacing spaces in Path inside a substring expression

### DISCUSSION THREAD

**Discussion by jdvelasquez-git** [microsoft-search/pnp-modern-search#2882] [Q&A]
Hi community,

I installed PnP Search for one of our SharePoint team sites and chose the adaptive cards layout.

Given that _path_ information is unavailable by default in this layout and that a specific team needs it when skimming search results, I added it to the template.

I tried to make their lives easier by removing the domain and library parts of the path using the `substring` expression, but it converts spaces to HTML.

`"text": "<strong>${substring(string(jPath($data, concat('.',$root.slots['Path']))[0]),94,99)}</strong>"`

How can I insert a `replace` expression that replaces the HTML spaces for "normal" spaces? 😊

Thank you in advance for all your help,
Joshua

**Comment by kasperbolarsen** [comment]
The Replace function should be available for Adaptive Cards : https://learn.microsoft.com/en-us/azure/bot-service/adaptive-expressions/adaptive-expressions-prebuilt-functions?view=azure-bot-service-4.0#replace
